safety/scan/command.py

```
@@ -246,6 +246,9 @@ def scan(ctx: typer.Context,
     """
     Scans a project (defaulted to the current directory) for supply-chain security and configuration issues
     """
+    
+    if ctx.obj.metadata.authenticated != True:
```

Ruff

Avoid inequality comparisons to `True`; use `if not ctx.obj.metadata.authenticated:` for false checks

Replace with `not ctx.obj.metadata.authenticated`

(E712)